Index field type polling job throws errors on shutdown with ES 7

## Description

I am seeing the following errors on server shutdown from time to time:

```
2020-08-14 11:46:21,325 ERROR: org.graylog2.indexer.fieldtypes.IndexFieldTypePollerPeriodical - Couldn't update field types for index set <watermark/5d2850f57e8acd3a598da539>
org.graylog.shaded.elasticsearch7.org.elasticsearch.ElasticsearchException: An error occurred: 
	at org.graylog.storage.elasticsearch7.ElasticsearchClient.exceptionFrom(ElasticsearchClient.java:97) ~[classes/:?]
	at org.graylog.storage.elasticsearch7.ElasticsearchClient.execute(ElasticsearchClient.java:72) ~[classes/:?]
	at org.graylog.storage.elasticsearch7.ElasticsearchClient.execute(ElasticsearchClient.java:65) ~[classes/:?]
	at org.graylog.storage.elasticsearch7.IndicesAdapterES7.resolveAlias(IndicesAdapterES7.java:119) ~[classes/:?]
	at org.graylog2.indexer.indices.Indices.aliasTarget(Indices.java:146) ~[classes/:?]
	at org.graylog2.indexer.MongoIndexSet.getActiveWriteIndex(MongoIndexSet.java:204) ~[classes/:?]
	at org.graylog2.indexer.fieldtypes.IndexFieldTypePollerPeriodical.lambda$schedule$4(IndexFieldTypePollerPeriodical.java:201) ~[classes/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_265]
	at java.util.concurrent.FutureTask.runAndReset$$$capture(FutureTask.java:308) [?:1.8.0_265]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java) [?:1.8.0_265]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_265]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) [?:1.8.0_265]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_265]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_265]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_265]
Caused by: java.lang.RuntimeException: Request cannot be executed; I/O reactor status: STOPPED
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestClient.extractAndWrapCause(RestClient.java:831) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestClient.performRequest(RestClient.java:248) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestClient.performRequest(RestClient.java:235) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.internalPerformRequest(RestHighLevelClient.java:1611) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.performRequest(RestHighLevelClient.java:1581) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.performRequestAndParseEntity(RestHighLevelClient.java:1551) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.IndicesClient.getAlias(IndicesClient.java:1175) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.storage.elasticsearch7.IndicesAdapterES7.lambda$resolveAlias$2(IndicesAdapterES7.java:119) ~[classes/:?]
	at org.graylog.storage.elasticsearch7.ElasticsearchClient.execute(ElasticsearchClient.java:70) ~[classes/:?]
	... 13 more
```

Here is the full server shutdown log with a few of these errors:

<details>
<summary>Full server shutdown log</summary>

```
2020-08-14 11:46:18,775 INFO : org.graylog2.commands.Server - SIGNAL received. Shutting down.
2020-08-14 11:46:18,784 INFO : org.graylog2.system.shutdown.GracefulShutdown - Graceful shutdown initiated.
2020-08-14 11:46:18,784 INFO : org.graylog2.system.shutdown.GracefulShutdown - Node status: [Halting [LB:DEAD]]. Waiting <0sec> for possible load balancers to recognize state change.
2020-08-14 11:46:19,786 INFO : org.graylog2.shared.initializers.InputSetupService - Attempting to close input <org.graylog2.inputs.gelf.tcp.GELFTCPInput.56f2fdefa0275b357744230c> [GELF TCP].
2020-08-14 11:46:19,799 INFO : org.graylog2.shared.initializers.InputSetupService - Input <org.graylog2.inputs.gelf.tcp.GELFTCPInput.56f2fdefa0275b357744230c> closed. Took [12ms]
2020-08-14 11:46:19,799 INFO : org.graylog2.shared.initializers.InputSetupService - Attempting to close input <org.graylog.integrations.inputs.paloalto.PaloAltoTCPInput.5bffdc107d8d924bf40f52d7> [Palo Alto Networks TCP (PAN-OS v8.x)].
2020-08-14 11:46:19,806 INFO : org.graylog2.shared.initializers.InputSetupService - Input <org.graylog.integrations.inputs.paloalto.PaloAltoTCPInput.5bffdc107d8d924bf40f52d7> closed. Took [6ms]
2020-08-14 11:46:19,806 INFO : org.graylog2.shared.initializers.InputSetupService - Attempting to close input <org.graylog2.inputs.syslog.tcp.SyslogTCPInput.5c9b9ab4d2b6ce0f99745f22> [Syslog TCP].
2020-08-14 11:46:19,809 INFO : org.graylog2.shared.initializers.InputSetupService - Input <org.graylog2.inputs.syslog.tcp.SyslogTCPInput.5c9b9ab4d2b6ce0f99745f22> closed. Took [2ms]
2020-08-14 11:46:19,810 INFO : org.graylog2.shared.initializers.InputSetupService - Attempting to close input <org.graylog2.inputs.syslog.udp.SyslogUDPInput.5b5acb170d7d705813e9c103> [Syslog UDP].
2020-08-14 11:46:19,813 INFO : org.graylog2.shared.initializers.InputSetupService - Input <org.graylog2.inputs.syslog.udp.SyslogUDPInput.5b5acb170d7d705813e9c103> closed. Took [3ms]
2020-08-14 11:46:19,813 INFO : org.graylog2.shared.initializers.InputSetupService - Attempting to close input <org.graylog.plugins.beats.BeatsInput.5bc47f6ad5515e035c623f66> [Beats (deprecated)].
2020-08-14 11:46:19,826 INFO : org.graylog2.shared.initializers.InputSetupService - Input <org.graylog.plugins.beats.BeatsInput.5bc47f6ad5515e035c623f66> closed. Took [13ms]
2020-08-14 11:46:19,826 INFO : org.graylog2.shared.initializers.InputSetupService - Attempting to close input <org.graylog2.inputs.raw.udp.RawUDPInput.5bf1a5e8dde14c63cd59a410> [Raw/Plaintext UDP].
2020-08-14 11:46:19,829 INFO : org.graylog2.shared.initializers.InputSetupService - Input <org.graylog2.inputs.raw.udp.RawUDPInput.5bf1a5e8dde14c63cd59a410> closed. Took [2ms]
2020-08-14 11:46:19,829 INFO : org.graylog2.shared.initializers.InputSetupService - Attempting to close input <org.graylog2.inputs.gelf.http.GELFHttpInput.57ab37cc67d0cb54582d43a0> [GELF HTTP].
2020-08-14 11:46:19,834 INFO : org.graylog2.shared.initializers.InputSetupService - Input <org.graylog2.inputs.gelf.http.GELFHttpInput.57ab37cc67d0cb54582d43a0> closed. Took [5ms]
2020-08-14 11:46:19,835 INFO : org.graylog2.shared.initializers.InputSetupService - Attempting to close input <org.graylog2.inputs.raw.tcp.RawTCPInput.5787be36326f19227288ab3c> [Raw/Plaintext TCP].
2020-08-14 11:46:19,839 INFO : org.graylog2.shared.initializers.InputSetupService - Input <org.graylog2.inputs.raw.tcp.RawTCPInput.5787be36326f19227288ab3c> closed. Took [4ms]
2020-08-14 11:46:19,839 INFO : org.graylog2.shared.initializers.InputSetupService - Attempting to close input <org.graylog2.inputs.random.FakeHttpMessageInput.5d2851367e8acd3a598da58b> [Random HTTP message generator].
2020-08-14 11:46:19,840 INFO : org.graylog2.shared.initializers.InputSetupService - Input <org.graylog2.inputs.random.FakeHttpMessageInput.5d2851367e8acd3a598da58b> closed. Took [0ms]
2020-08-14 11:46:19,840 INFO : org.graylog2.shared.initializers.InputSetupService - Attempting to close input <org.graylog.plugins.beats.Beats2Input.5b17e2653f3ab8204eea1125> [Beats].
2020-08-14 11:46:19,845 INFO : org.graylog2.shared.initializers.InputSetupService - Input <org.graylog.plugins.beats.Beats2Input.5b17e2653f3ab8204eea1125> closed. Took [5ms]
2020-08-14 11:46:19,846 INFO : org.graylog2.shared.initializers.InputSetupService - Attempting to close input <org.graylog2.inputs.gelf.udp.GELFUDPInput.56681558c8306c288fc5cc77> [GELF UDP].
2020-08-14 11:46:19,847 INFO : org.graylog2.shared.initializers.InputSetupService - Input <org.graylog2.inputs.gelf.udp.GELFUDPInput.56681558c8306c288fc5cc77> closed. Took [1ms]
2020-08-14 11:46:19,847 INFO : org.graylog2.shared.initializers.InputSetupService - Attempting to close input <org.graylog2.inputs.raw.tcp.RawTCPInput.593e57acfb499259c23e4371> [Raw/Plaintext TCP].
2020-08-14 11:46:19,854 INFO : org.graylog2.shared.initializers.InputSetupService - Input <org.graylog2.inputs.raw.tcp.RawTCPInput.593e57acfb499259c23e4371> closed. Took [6ms]
2020-08-14 11:46:19,854 INFO : org.graylog2.shared.initializers.InputSetupService - Attempting to close input <org.graylog2.inputs.raw.tcp.RawTCPInput.5c1923f63b08a31570fc9cc7> [Raw/Plaintext TCP].
2020-08-14 11:46:19,861 INFO : org.graylog2.shared.initializers.InputSetupService - Input <org.graylog2.inputs.raw.tcp.RawTCPInput.5c1923f63b08a31570fc9cc7> closed. Took [6ms]
2020-08-14 11:46:19,861 INFO : org.graylog2.shared.initializers.InputSetupService - Attempting to close input <org.graylog2.inputs.raw.tcp.RawTCPInput.5e174e17d4714e5c724e69ef> [Raw/Plaintext TCP].
2020-08-14 11:46:19,869 INFO : org.graylog2.shared.initializers.InputSetupService - Input <org.graylog2.inputs.raw.tcp.RawTCPInput.5e174e17d4714e5c724e69ef> closed. Took [8ms]
2020-08-14 11:46:19,870 INFO : org.graylog2.shared.initializers.InputSetupService - Attempting to close input <org.graylog2.inputs.raw.tcp.RawTCPInput.564dcd12c830e40a3ba4b743> [Raw/Plaintext TCP].
2020-08-14 11:46:19,878 INFO : org.graylog2.shared.initializers.InputSetupService - Input <org.graylog2.inputs.raw.tcp.RawTCPInput.564dcd12c830e40a3ba4b743> closed. Took [8ms]
2020-08-14 11:46:19,878 INFO : org.graylog2.shared.initializers.InputSetupService - Attempting to close input <org.graylog.plugins.netflow.inputs.NetFlowUdpInput.599f04e43cf729735fe8f587> [NetFlow UDP].
2020-08-14 11:46:19,880 INFO : org.graylog2.shared.initializers.InputSetupService - Input <org.graylog.plugins.netflow.inputs.NetFlowUdpInput.599f04e43cf729735fe8f587> closed. Took [1ms]
2020-08-14 11:46:19,900 INFO : org.graylog2.buffers.Buffers - Waiting until all buffers are empty.
2020-08-14 11:46:19,901 INFO : org.graylog2.buffers.Buffers - All buffers are empty. Continuing.
2020-08-14 11:46:19,901 INFO : org.graylog2.initializers.OutputSetupService - Stopping output org.graylog2.outputs.BlockingBatchedESOutput
2020-08-14 11:46:19,902 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutting down periodical [org.graylog2.periodical.BatchedElasticSearchOutputFlushThread].
2020-08-14 11:46:19,902 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutdown of periodical [org.graylog2.periodical.BatchedElasticSearchOutputFlushThread] complete, took <0ms>.
2020-08-14 11:46:19,902 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutting down periodical [org.graylog2.periodical.ClusterHealthCheckThread].
2020-08-14 11:46:19,902 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutdown of periodical [org.graylog2.periodical.ClusterHealthCheckThread] complete, took <0ms>.
2020-08-14 11:46:19,902 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutting down periodical [org.graylog2.periodical.IndexerClusterCheckerThread].
2020-08-14 11:46:19,902 INFO : org.graylog2.system.shutdown.GracefulShutdownService - Running graceful shutdown for <3> shutdown hooks
2020-08-14 11:46:19,902 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutdown of periodical [org.graylog2.periodical.IndexerClusterCheckerThread] complete, took <0ms>.
2020-08-14 11:46:19,902 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutting down periodical [org.graylog2.periodical.IndexRetentionThread].
2020-08-14 11:46:19,902 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutdown of periodical [org.graylog2.periodical.IndexRetentionThread] complete, took <0ms>.
2020-08-14 11:46:19,903 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutting down periodical [org.graylog2.periodical.IndexRotationThread].
2020-08-14 11:46:19,903 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutdown of periodical [org.graylog2.periodical.IndexRotationThread] complete, took <0ms>.
2020-08-14 11:46:19,903 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutting down periodical [org.graylog2.periodical.ThrottleStateUpdaterThread].
2020-08-14 11:46:19,903 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutdown of periodical [org.graylog2.periodical.ThrottleStateUpdaterThread] complete, took <0ms>.
2020-08-14 11:46:19,903 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutting down periodical [org.graylog2.events.ClusterEventPeriodical].
2020-08-14 11:46:19,903 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutdown of periodical [org.graylog2.events.ClusterEventPeriodical] complete, took <0ms>.
2020-08-14 11:46:19,903 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutting down periodical [org.graylog2.events.ClusterEventCleanupPeriodical].
2020-08-14 11:46:19,903 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutdown of periodical [org.graylog2.events.ClusterEventCleanupPeriodical] complete, took <0ms>.
2020-08-14 11:46:19,903 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutting down periodical [org.graylog2.periodical.IndexRangesCleanupPeriodical].
2020-08-14 11:46:19,903 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutdown of periodical [org.graylog2.periodical.IndexRangesCleanupPeriodical] complete, took <0ms>.
2020-08-14 11:46:19,903 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutting down periodical [org.graylog2.periodical.TrafficCounterCalculator].
2020-08-14 11:46:19,903 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutdown of periodical [org.graylog2.periodical.TrafficCounterCalculator] complete, took <0ms>.
2020-08-14 11:46:19,903 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutting down periodical [org.graylog2.indexer.fieldtypes.IndexFieldTypePollerPeriodical].
2020-08-14 11:46:19,903 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutdown of periodical [org.graylog2.indexer.fieldtypes.IndexFieldTypePollerPeriodical] complete, took <0ms>.
2020-08-14 11:46:19,903 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutting down periodical [org.graylog.scheduler.periodicals.ScheduleTriggerCleanUp].
2020-08-14 11:46:19,903 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutdown of periodical [org.graylog.scheduler.periodicals.ScheduleTriggerCleanUp] complete, took <0ms>.
2020-08-14 11:46:19,903 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutting down periodical [org.graylog.plugins.sidecar.periodical.PurgeExpiredSidecarsThread].
2020-08-14 11:46:19,903 INFO : org.graylog2.system.shutdown.GracefulShutdownService - Initiate shutdown for <BrowserDriverService>
2020-08-14 11:46:19,903 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutdown of periodical [org.graylog.plugins.sidecar.periodical.PurgeExpiredSidecarsThread] complete, took <0ms>.
2020-08-14 11:46:19,903 INFO : org.graylog2.system.shutdown.GracefulShutdownService - Initiate shutdown for <RestHighLevelClientProvider$$Lambda$282/193206035>
2020-08-14 11:46:19,903 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutting down periodical [org.graylog.plugins.sidecar.periodical.PurgeExpiredConfigurationUploads].
2020-08-14 11:46:19,903 INFO : org.graylog2.system.shutdown.GracefulShutdownService - Finished shutdown for <BrowserDriverService>, took 0 ms
2020-08-14 11:46:19,903 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutdown of periodical [org.graylog.plugins.sidecar.periodical.PurgeExpiredConfigurationUploads] complete, took <0ms>.
2020-08-14 11:46:19,903 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutting down periodical [org.graylog.plugins.views.search.db.SearchesCleanUpJob].
2020-08-14 11:46:19,904 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutdown of periodical [org.graylog.plugins.views.search.db.SearchesCleanUpJob] complete, took <0ms>.
2020-08-14 11:46:19,904 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutting down periodical [org.graylog.events.periodicals.EventNotificationStatusCleanUp].
2020-08-14 11:46:19,904 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutdown of periodical [org.graylog.events.periodicals.EventNotificationStatusCleanUp] complete, took <0ms>.
2020-08-14 11:46:19,904 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutting down periodical [org.graylog.plugins.collector.periodical.PurgeExpiredCollectorsThread].
2020-08-14 11:46:19,904 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutdown of periodical [org.graylog.plugins.collector.periodical.PurgeExpiredCollectorsThread] complete, took <0ms>.
2020-08-14 11:46:19,904 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutting down periodical [org.graylog.plugins.license.LicenseManagerPeriodical].
2020-08-14 11:46:19,904 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutdown of periodical [org.graylog.plugins.license.LicenseManagerPeriodical] complete, took <0ms>.
2020-08-14 11:46:19,904 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutting down periodical [org.graylog.plugins.license.LicenseReportPeriodical].
2020-08-14 11:46:19,904 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutdown of periodical [org.graylog.plugins.license.LicenseReportPeriodical] complete, took <0ms>.
2020-08-14 11:46:19,904 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutting down periodical [org.graylog.plugins.report.scheduler.ReportPeriodical].
2020-08-14 11:46:19,904 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutdown of periodical [org.graylog.plugins.report.scheduler.ReportPeriodical] complete, took <0ms>.
2020-08-14 11:46:19,904 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutting down periodical [org.graylog.plugins.report.service.ChromeDriverCleanupPeriodical].
2020-08-14 11:46:19,904 INFO : org.graylog2.shared.initializers.PeriodicalsService - Shutdown of periodical [org.graylog.plugins.report.service.ChromeDriverCleanupPeriodical] complete, took <0ms>.
2020-08-14 11:46:19,910 INFO : org.graylog2.system.shutdown.GracefulShutdownService - Initiate shutdown for <JobWorkerPool>
2020-08-14 11:46:19,928 INFO : org.graylog2.system.shutdown.GracefulShutdownService - Finished shutdown for <JobWorkerPool>, took 17 ms
2020-08-14 11:46:19,934 INFO : org.graylog2.system.shutdown.GracefulShutdownService - Finished shutdown for <RestHighLevelClientProvider$$Lambda$282/193206035>, took 30 ms
2020-08-14 11:46:19,938 INFO : org.graylog2.system.shutdown.GracefulShutdown - Goodbye.
2020-08-14 11:46:19,939 INFO : org.graylog2.shared.initializers.JerseyService - Shutting down HTTP listener at <127.0.0.1:9000>
2020-08-14 11:46:19,946 INFO : org.graylog2.lookup.LookupTableService - Cache in-memory/59033c5e8ae492261e064b17 [@7d105e6] STOPPING, was RUNNING
2020-08-14 11:46:19,946 INFO : org.graylog2.lookup.LookupTableService - Cache in-memory/59033c5e8ae492261e064b17 [@7d105e6] TERMINATED, was STOPPING
2020-08-14 11:46:19,963 INFO : org.graylog2.lookup.LookupTableService - Cache noop-cache/5909830701763e2c4806766c [@9a42fbd] STOPPING, was RUNNING
2020-08-14 11:46:19,964 INFO : org.graylog2.lookup.LookupTableService - Cache noop-cache/5909830701763e2c4806766c [@9a42fbd] TERMINATED, was STOPPING
2020-08-14 11:46:19,972 INFO : kafka.log.LogManager - Shutting down.
2020-08-14 11:46:19,981 INFO : org.graylog2.lookup.LookupTableService - Data Adapter kv-2/5e21d0ec65da5d14e8b398b7 [@735e7b16] STOPPING, was RUNNING
2020-08-14 11:46:19,981 INFO : org.graylog2.lookup.LookupTableService - Data Adapter kv-2/5e21d0ec65da5d14e8b398b7 [@735e7b16] TERMINATED, was STOPPING
2020-08-14 11:46:19,984 INFO : org.graylog2.lookup.LookupDataAdapterRefreshService - Removing job for <maxmind-asn/5daf022e43ee214c237c8143/@7f07a72e>
2020-08-14 11:46:19,984 INFO : org.graylog2.lookup.LookupTableService - Data Adapter maxmind-asn/5daf022e43ee214c237c8143 [@7f07a72e] STOPPING, was RUNNING
2020-08-14 11:46:19,984 INFO : org.graylog2.lookup.LookupTableService - Data Adapter maxmind-asn/5daf022e43ee214c237c8143 [@7f07a72e] TERMINATED, was STOPPING
2020-08-14 11:46:19,986 INFO : org.graylog2.lookup.LookupTableService - Data Adapter nocsv/593a6dee9e189127752547fe [@3567dbb0] STOPPING, was RUNNING
2020-08-14 11:46:19,986 INFO : org.graylog2.lookup.LookupDataAdapterRefreshService - Removing job for <nocsv/593a6dee9e189127752547fe/@3567dbb0>
2020-08-14 11:46:19,986 INFO : org.graylog2.lookup.LookupTableService - Data Adapter nocsv/593a6dee9e189127752547fe [@3567dbb0] TERMINATED, was STOPPING
2020-08-14 11:46:19,995 INFO : org.graylog2.lookup.LookupTableService - Data Adapter csv/58ff051001763e1562d5adc9 [@65b501cd] STOPPING, was RUNNING
2020-08-14 11:46:19,995 INFO : org.graylog2.lookup.LookupTableService - Data Adapter csv/58ff051001763e1562d5adc9 [@65b501cd] TERMINATED, was STOPPING
2020-08-14 11:46:19,995 INFO : org.graylog2.lookup.LookupDataAdapterRefreshService - Removing job for <csv/58ff051001763e1562d5adc9/@65b501cd>
2020-08-14 11:46:19,996 INFO : org.graylog2.lookup.LookupDataAdapterRefreshService - Removing job for <maxmind-country/5a6714a5c1a47b2f18d4dcc4/@6f2b597e>
2020-08-14 11:46:19,996 INFO : org.graylog2.lookup.LookupTableService - Data Adapter maxmind-country/5a6714a5c1a47b2f18d4dcc4 [@6f2b597e] STOPPING, was RUNNING
2020-08-14 11:46:19,996 INFO : org.graylog2.lookup.LookupTableService - Data Adapter maxmind-country/5a6714a5c1a47b2f18d4dcc4 [@6f2b597e] TERMINATED, was STOPPING
2020-08-14 11:46:19,998 INFO : org.graylog2.lookup.LookupTableService - Data Adapter kv/5dd6f79f9e38332c92ffdfa7 [@7f1b970d] STOPPING, was RUNNING
2020-08-14 11:46:19,998 INFO : org.graylog2.lookup.LookupTableService - Data Adapter kv/5dd6f79f9e38332c92ffdfa7 [@7f1b970d] TERMINATED, was STOPPING
2020-08-14 11:46:20,004 INFO : org.graylog2.lookup.LookupTableService - Data Adapter lladdr/595a5fa01162612bb4c8d1d6 [@1b7e56eb] STOPPING, was RUNNING
2020-08-14 11:46:20,004 INFO : org.graylog2.lookup.LookupDataAdapterRefreshService - Removing job for <lladdr/595a5fa01162612bb4c8d1d6/@1b7e56eb>
2020-08-14 11:46:20,004 INFO : org.graylog2.lookup.LookupTableService - Data Adapter lladdr/595a5fa01162612bb4c8d1d6 [@1b7e56eb] TERMINATED, was STOPPING
2020-08-14 11:46:20,023 INFO : org.graylog2.lookup.LookupTableService - Data Adapter dns-a-over-https/5908bc2601763e1460e465fe [@46ffa324] STOPPING, was RUNNING
2020-08-14 11:46:20,023 INFO : org.graylog2.lookup.LookupTableService - Data Adapter http-verbs/5df285f4cb9786112e480bc6 [@d91d16c] STOPPING, was RUNNING
2020-08-14 11:46:20,029 INFO : org.graylog2.lookup.LookupTableService - Data Adapter dns-lookup/5c18d3723b08a31570fc4578 [@327173c4] STOPPING, was RUNNING
2020-08-14 11:46:20,026 INFO : org.graylog2.lookup.LookupTableService - Data Adapter maxmind/5909ad968ae49210ae6c6897 [@2e4721f7] STOPPING, was RUNNING
2020-08-14 11:46:20,029 INFO : org.graylog2.shared.journal.JournalReader - Stopping.
2020-08-14 11:46:20,029 INFO : org.graylog2.lookup.LookupTableService - Data Adapter http-verbs/5df285f4cb9786112e480bc6 [@d91d16c] TERMINATED, was STOPPING
2020-08-14 11:46:20,030 INFO : org.graylog2.lookup.LookupTableService - Data Adapter dns-a-over-https/5908bc2601763e1460e465fe [@46ffa324] TERMINATED, was STOPPING
2020-08-14 11:46:20,030 INFO : org.graylog2.lookup.LookupDataAdapterRefreshService - Stopping 1 jobs
2020-08-14 11:46:20,030 INFO : org.graylog2.lookup.LookupTableService - Data Adapter maxmind/5909ad968ae49210ae6c6897 [@2e4721f7] TERMINATED, was STOPPING
2020-08-14 11:46:20,043 INFO : kafka.log.LogManager - Shutdown complete.
2020-08-14 11:46:20,078 INFO : org.graylog2.lookup.LookupTableService - Data Adapter dns-lookup/5c18d3723b08a31570fc4578 [@327173c4] TERMINATED, was STOPPING
2020-08-14 11:46:21,144 ERROR: org.graylog2.indexer.fieldtypes.IndexFieldTypePollerPeriodical - Couldn't update field types for index set <Default index set/582f3c4e43761335e4859f72>
org.graylog.shaded.elasticsearch7.org.elasticsearch.ElasticsearchException: An error occurred: 
	at org.graylog.storage.elasticsearch7.ElasticsearchClient.exceptionFrom(ElasticsearchClient.java:97) ~[classes/:?]
	at org.graylog.storage.elasticsearch7.ElasticsearchClient.execute(ElasticsearchClient.java:72) ~[classes/:?]
	at org.graylog.storage.elasticsearch7.ElasticsearchClient.execute(ElasticsearchClient.java:65) ~[classes/:?]
	at org.graylog.storage.elasticsearch7.IndicesAdapterES7.resolveAlias(IndicesAdapterES7.java:119) ~[classes/:?]
	at org.graylog2.indexer.indices.Indices.aliasTarget(Indices.java:146) ~[classes/:?]
	at org.graylog2.indexer.MongoIndexSet.getActiveWriteIndex(MongoIndexSet.java:204) ~[classes/:?]
	at org.graylog2.indexer.fieldtypes.IndexFieldTypePollerPeriodical.lambda$schedule$4(IndexFieldTypePollerPeriodical.java:201) ~[classes/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_265]
	at java.util.concurrent.FutureTask.runAndReset$$$capture(FutureTask.java:308) [?:1.8.0_265]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java) [?:1.8.0_265]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_265]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) [?:1.8.0_265]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_265]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_265]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_265]
Caused by: java.lang.RuntimeException: Request cannot be executed; I/O reactor status: STOPPED
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestClient.extractAndWrapCause(RestClient.java:831) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestClient.performRequest(RestClient.java:248) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestClient.performRequest(RestClient.java:235) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.internalPerformRequest(RestHighLevelClient.java:1611) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.performRequest(RestHighLevelClient.java:1581) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.performRequestAndParseEntity(RestHighLevelClient.java:1551) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.IndicesClient.getAlias(IndicesClient.java:1175) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.storage.elasticsearch7.IndicesAdapterES7.lambda$resolveAlias$2(IndicesAdapterES7.java:119) ~[classes/:?]
	at org.graylog.storage.elasticsearch7.ElasticsearchClient.execute(ElasticsearchClient.java:70) ~[classes/:?]
	... 13 more
Caused by: java.lang.IllegalStateException: Request cannot be executed; I/O reactor status: STOPPED
	at org.graylog.shaded.elasticsearch7.org.apache.http.util.Asserts.check(Asserts.java:46) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.apache.http.impl.nio.client.CloseableHttpAsyncClientBase.ensureRunning(CloseableHttpAsyncClientBase.java:90) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.apache.http.impl.nio.client.InternalHttpAsyncClient.execute(InternalHttpAsyncClient.java:123) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestClient.performRequest(RestClient.java:244) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestClient.performRequest(RestClient.java:235) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.internalPerformRequest(RestHighLevelClient.java:1611) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.performRequest(RestHighLevelClient.java:1581) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.performRequestAndParseEntity(RestHighLevelClient.java:1551) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.IndicesClient.getAlias(IndicesClient.java:1175) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.storage.elasticsearch7.IndicesAdapterES7.lambda$resolveAlias$2(IndicesAdapterES7.java:119) ~[classes/:?]
	at org.graylog.storage.elasticsearch7.ElasticsearchClient.execute(ElasticsearchClient.java:70) ~[classes/:?]
	... 13 more
2020-08-14 11:46:21,256 ERROR: org.graylog2.indexer.fieldtypes.IndexFieldTypePollerPeriodical - Couldn't update field types for index set <Yolo/584589ca2f857c1c2799a77e>
org.graylog.shaded.elasticsearch7.org.elasticsearch.ElasticsearchException: An error occurred: 
	at org.graylog.storage.elasticsearch7.ElasticsearchClient.exceptionFrom(ElasticsearchClient.java:97) ~[classes/:?]
	at org.graylog.storage.elasticsearch7.ElasticsearchClient.execute(ElasticsearchClient.java:72) ~[classes/:?]
	at org.graylog.storage.elasticsearch7.ElasticsearchClient.execute(ElasticsearchClient.java:65) ~[classes/:?]
	at org.graylog.storage.elasticsearch7.IndicesAdapterES7.resolveAlias(IndicesAdapterES7.java:119) ~[classes/:?]
	at org.graylog2.indexer.indices.Indices.aliasTarget(Indices.java:146) ~[classes/:?]
	at org.graylog2.indexer.MongoIndexSet.getActiveWriteIndex(MongoIndexSet.java:204) ~[classes/:?]
	at org.graylog2.indexer.fieldtypes.IndexFieldTypePollerPeriodical.lambda$schedule$4(IndexFieldTypePollerPeriodical.java:201) ~[classes/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_265]
	at java.util.concurrent.FutureTask.runAndReset$$$capture(FutureTask.java:308) [?:1.8.0_265]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java) [?:1.8.0_265]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_265]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) [?:1.8.0_265]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_265]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_265]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_265]
Caused by: java.lang.RuntimeException: Request cannot be executed; I/O reactor status: STOPPED
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestClient.extractAndWrapCause(RestClient.java:831) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestClient.performRequest(RestClient.java:248) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestClient.performRequest(RestClient.java:235) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.internalPerformRequest(RestHighLevelClient.java:1611) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.performRequest(RestHighLevelClient.java:1581) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.performRequestAndParseEntity(RestHighLevelClient.java:1551) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.IndicesClient.getAlias(IndicesClient.java:1175) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.storage.elasticsearch7.IndicesAdapterES7.lambda$resolveAlias$2(IndicesAdapterES7.java:119) ~[classes/:?]
	at org.graylog.storage.elasticsearch7.ElasticsearchClient.execute(ElasticsearchClient.java:70) ~[classes/:?]
	... 13 more
Caused by: java.lang.IllegalStateException: Request cannot be executed; I/O reactor status: STOPPED
	at org.graylog.shaded.elasticsearch7.org.apache.http.util.Asserts.check(Asserts.java:46) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.apache.http.impl.nio.client.CloseableHttpAsyncClientBase.ensureRunning(CloseableHttpAsyncClientBase.java:90) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.apache.http.impl.nio.client.InternalHttpAsyncClient.execute(InternalHttpAsyncClient.java:123) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestClient.performRequest(RestClient.java:244) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestClient.performRequest(RestClient.java:235) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.internalPerformRequest(RestHighLevelClient.java:1611) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.performRequest(RestHighLevelClient.java:1581) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.performRequestAndParseEntity(RestHighLevelClient.java:1551) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.IndicesClient.getAlias(IndicesClient.java:1175) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.storage.elasticsearch7.IndicesAdapterES7.lambda$resolveAlias$2(IndicesAdapterES7.java:119) ~[classes/:?]
	at org.graylog.storage.elasticsearch7.ElasticsearchClient.execute(ElasticsearchClient.java:70) ~[classes/:?]
	... 13 more
2020-08-14 11:46:21,281 ERROR: org.graylog2.indexer.fieldtypes.IndexFieldTypePollerPeriodical - Couldn't update field types for index set <issue-4954/5b59efb20d7d706805c2a108>
org.graylog.shaded.elasticsearch7.org.elasticsearch.ElasticsearchException: An error occurred: 
	at org.graylog.storage.elasticsearch7.ElasticsearchClient.exceptionFrom(ElasticsearchClient.java:97) ~[classes/:?]
	at org.graylog.storage.elasticsearch7.ElasticsearchClient.execute(ElasticsearchClient.java:72) ~[classes/:?]
	at org.graylog.storage.elasticsearch7.ElasticsearchClient.execute(ElasticsearchClient.java:65) ~[classes/:?]
	at org.graylog.storage.elasticsearch7.IndicesAdapterES7.resolveAlias(IndicesAdapterES7.java:119) ~[classes/:?]
	at org.graylog2.indexer.indices.Indices.aliasTarget(Indices.java:146) ~[classes/:?]
	at org.graylog2.indexer.MongoIndexSet.getActiveWriteIndex(MongoIndexSet.java:204) ~[classes/:?]
	at org.graylog2.indexer.fieldtypes.IndexFieldTypePollerPeriodical.lambda$schedule$4(IndexFieldTypePollerPeriodical.java:201) ~[classes/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_265]
	at java.util.concurrent.FutureTask.runAndReset$$$capture(FutureTask.java:308) [?:1.8.0_265]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java) [?:1.8.0_265]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_265]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) [?:1.8.0_265]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_265]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_265]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_265]
Caused by: java.lang.RuntimeException: Request cannot be executed; I/O reactor status: STOPPED
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestClient.extractAndWrapCause(RestClient.java:831) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestClient.performRequest(RestClient.java:248) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestClient.performRequest(RestClient.java:235) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.internalPerformRequest(RestHighLevelClient.java:1611) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.performRequest(RestHighLevelClient.java:1581) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.performRequestAndParseEntity(RestHighLevelClient.java:1551) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.IndicesClient.getAlias(IndicesClient.java:1175) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.storage.elasticsearch7.IndicesAdapterES7.lambda$resolveAlias$2(IndicesAdapterES7.java:119) ~[classes/:?]
	at org.graylog.storage.elasticsearch7.ElasticsearchClient.execute(ElasticsearchClient.java:70) ~[classes/:?]
	... 13 more
Caused by: java.lang.IllegalStateException: Request cannot be executed; I/O reactor status: STOPPED
	at org.graylog.shaded.elasticsearch7.org.apache.http.util.Asserts.check(Asserts.java:46) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.apache.http.impl.nio.client.CloseableHttpAsyncClientBase.ensureRunning(CloseableHttpAsyncClientBase.java:90) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.apache.http.impl.nio.client.InternalHttpAsyncClient.execute(InternalHttpAsyncClient.java:123) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestClient.performRequest(RestClient.java:244) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestClient.performRequest(RestClient.java:235) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.internalPerformRequest(RestHighLevelClient.java:1611) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.performRequest(RestHighLevelClient.java:1581) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.performRequestAndParseEntity(RestHighLevelClient.java:1551) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.IndicesClient.getAlias(IndicesClient.java:1175) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.storage.elasticsearch7.IndicesAdapterES7.lambda$resolveAlias$2(IndicesAdapterES7.java:119) ~[classes/:?]
	at org.graylog.storage.elasticsearch7.ElasticsearchClient.execute(ElasticsearchClient.java:70) ~[classes/:?]
	... 13 more
2020-08-14 11:46:21,300 ERROR: org.graylog2.indexer.fieldtypes.IndexFieldTypePollerPeriodical - Couldn't update field types for index set <oinkeria/5a983b8e84eafb2dde100801>
org.graylog.shaded.elasticsearch7.org.elasticsearch.ElasticsearchException: An error occurred: 
	at org.graylog.storage.elasticsearch7.ElasticsearchClient.exceptionFrom(ElasticsearchClient.java:97) ~[classes/:?]
	at org.graylog.storage.elasticsearch7.ElasticsearchClient.execute(ElasticsearchClient.java:72) ~[classes/:?]
	at org.graylog.storage.elasticsearch7.ElasticsearchClient.execute(ElasticsearchClient.java:65) ~[classes/:?]
	at org.graylog.storage.elasticsearch7.IndicesAdapterES7.resolveAlias(IndicesAdapterES7.java:119) ~[classes/:?]
	at org.graylog2.indexer.indices.Indices.aliasTarget(Indices.java:146) ~[classes/:?]
	at org.graylog2.indexer.MongoIndexSet.getActiveWriteIndex(MongoIndexSet.java:204) ~[classes/:?]
	at org.graylog2.indexer.fieldtypes.IndexFieldTypePollerPeriodical.lambda$schedule$4(IndexFieldTypePollerPeriodical.java:201) ~[classes/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_265]
	at java.util.concurrent.FutureTask.runAndReset$$$capture(FutureTask.java:308) [?:1.8.0_265]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java) [?:1.8.0_265]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_265]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) [?:1.8.0_265]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_265]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_265]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_265]
Caused by: java.lang.RuntimeException: Request cannot be executed; I/O reactor status: STOPPED
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestClient.extractAndWrapCause(RestClient.java:831) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestClient.performRequest(RestClient.java:248) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestClient.performRequest(RestClient.java:235) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.internalPerformRequest(RestHighLevelClient.java:1611) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.performRequest(RestHighLevelClient.java:1581) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.performRequestAndParseEntity(RestHighLevelClient.java:1551) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.IndicesClient.getAlias(IndicesClient.java:1175) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.storage.elasticsearch7.IndicesAdapterES7.lambda$resolveAlias$2(IndicesAdapterES7.java:119) ~[classes/:?]
	at org.graylog.storage.elasticsearch7.ElasticsearchClient.execute(ElasticsearchClient.java:70) ~[classes/:?]
	... 13 more
Caused by: java.lang.IllegalStateException: Request cannot be executed; I/O reactor status: STOPPED
	at org.graylog.shaded.elasticsearch7.org.apache.http.util.Asserts.check(Asserts.java:46) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.apache.http.impl.nio.client.CloseableHttpAsyncClientBase.ensureRunning(CloseableHttpAsyncClientBase.java:90) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.apache.http.impl.nio.client.InternalHttpAsyncClient.execute(InternalHttpAsyncClient.java:123) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestClient.performRequest(RestClient.java:244) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestClient.performRequest(RestClient.java:235) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.internalPerformRequest(RestHighLevelClient.java:1611) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.performRequest(RestHighLevelClient.java:1581) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.performRequestAndParseEntity(RestHighLevelClient.java:1551) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.IndicesClient.getAlias(IndicesClient.java:1175) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.storage.elasticsearch7.IndicesAdapterES7.lambda$resolveAlias$2(IndicesAdapterES7.java:119) ~[classes/:?]
	at org.graylog.storage.elasticsearch7.ElasticsearchClient.execute(ElasticsearchClient.java:70) ~[classes/:?]
	... 13 more
2020-08-14 11:46:21,325 ERROR: org.graylog2.indexer.fieldtypes.IndexFieldTypePollerPeriodical - Couldn't update field types for index set <watermark/5d2850f57e8acd3a598da539>
org.graylog.shaded.elasticsearch7.org.elasticsearch.ElasticsearchException: An error occurred: 
	at org.graylog.storage.elasticsearch7.ElasticsearchClient.exceptionFrom(ElasticsearchClient.java:97) ~[classes/:?]
	at org.graylog.storage.elasticsearch7.ElasticsearchClient.execute(ElasticsearchClient.java:72) ~[classes/:?]
	at org.graylog.storage.elasticsearch7.ElasticsearchClient.execute(ElasticsearchClient.java:65) ~[classes/:?]
	at org.graylog.storage.elasticsearch7.IndicesAdapterES7.resolveAlias(IndicesAdapterES7.java:119) ~[classes/:?]
	at org.graylog2.indexer.indices.Indices.aliasTarget(Indices.java:146) ~[classes/:?]
	at org.graylog2.indexer.MongoIndexSet.getActiveWriteIndex(MongoIndexSet.java:204) ~[classes/:?]
	at org.graylog2.indexer.fieldtypes.IndexFieldTypePollerPeriodical.lambda$schedule$4(IndexFieldTypePollerPeriodical.java:201) ~[classes/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_265]
	at java.util.concurrent.FutureTask.runAndReset$$$capture(FutureTask.java:308) [?:1.8.0_265]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java) [?:1.8.0_265]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_265]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) [?:1.8.0_265]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_265]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_265]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_265]
Caused by: java.lang.RuntimeException: Request cannot be executed; I/O reactor status: STOPPED
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestClient.extractAndWrapCause(RestClient.java:831) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestClient.performRequest(RestClient.java:248) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestClient.performRequest(RestClient.java:235) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.internalPerformRequest(RestHighLevelClient.java:1611) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.performRequest(RestHighLevelClient.java:1581) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.performRequestAndParseEntity(RestHighLevelClient.java:1551) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.IndicesClient.getAlias(IndicesClient.java:1175) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.storage.elasticsearch7.IndicesAdapterES7.lambda$resolveAlias$2(IndicesAdapterES7.java:119) ~[classes/:?]
	at org.graylog.storage.elasticsearch7.ElasticsearchClient.execute(ElasticsearchClient.java:70) ~[classes/:?]
	... 13 more
Caused by: java.lang.IllegalStateException: Request cannot be executed; I/O reactor status: STOPPED
	at org.graylog.shaded.elasticsearch7.org.apache.http.util.Asserts.check(Asserts.java:46) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.apache.http.impl.nio.client.CloseableHttpAsyncClientBase.ensureRunning(CloseableHttpAsyncClientBase.java:90) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.apache.http.impl.nio.client.InternalHttpAsyncClient.execute(InternalHttpAsyncClient.java:123) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestClient.performRequest(RestClient.java:244) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestClient.performRequest(RestClient.java:235) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.internalPerformRequest(RestHighLevelClient.java:1611) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.performRequest(RestHighLevelClient.java:1581) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.RestHighLevelClient.performRequestAndParseEntity(RestHighLevelClient.java:1551) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.shaded.elasticsearch7.org.elasticsearch.client.IndicesClient.getAlias(IndicesClient.java:1175) ~[elasticsearch7-7.8.1-0.jar:?]
	at org.graylog.storage.elasticsearch7.IndicesAdapterES7.lambda$resolveAlias$2(IndicesAdapterES7.java:119) ~[classes/:?]
	at org.graylog.storage.elasticsearch7.ElasticsearchClient.execute(ElasticsearchClient.java:70) ~[classes/:?]
	... 13 more
2020-08-14 11:46:23,207 INFO : org.glassfish.grizzly.http.server.NetworkListener - Stopped listener bound to [127.0.0.1:9000]
2020-08-14 11:46:23,209 INFO : org.graylog2.shared.initializers.ServiceManagerListener - Services are now stopped.

Process finished with exit code 130 (interrupted by signal 2: SIGINT)
```

</details>

## Possible Solution

The `IndexFieldTypePoller` should probably take the server status into account so it doesn't try to update the field types during server shutdown.

## Steps to Reproduce (for bugs)

1. Start and stop the server a few times until you hit the point where the index field type polling job is running
2. Check error logs in the server log

## Your Environment

* Graylog Version: 4.0.0-SNAPSHOT
* Java Version: 1.8.0_265
* Elasticsearch Version: 7.8.1
* MongoDB Version: 3.6

--- a/graylog2-server/src/main/java/org/graylog2/indexer/fieldtypes/IndexFieldTypePollerPeriodical.java
+++ b/graylog2-server/src/main/java/org/graylog2/indexer/fieldtypes/IndexFieldTypePollerPeriodical.java
@@ -30,6 +30,8 @@ import org.graylog2.indexer.indexset.events.IndexSetDeletedEvent;
 import org.graylog2.indexer.indices.Indices;
 import org.graylog2.indexer.indices.TooManyAliasesException;
 import org.graylog2.indexer.indices.events.IndicesDeletedEvent;
+import org.graylog2.plugin.ServerStatus;
+import org.graylog2.plugin.lifecycles.Lifecycle;
 import org.graylog2.plugin.periodical.Periodical;
 import org.joda.time.Duration;
 import org.slf4j.Logger;
@@ -59,6 +61,7 @@ public class IndexFieldTypePollerPeriodical extends Periodical {
     private final Indices indices;
     private final MongoIndexSet.Factory mongoIndexSetFactory;
     private final Cluster cluster;
+    private final ServerStatus serverStatus;
     private final com.github.joschi.jadconfig.util.Duration periodicalInterval;
     private final ScheduledExecutorService scheduler;
     private final ConcurrentMap<String, ScheduledFuture<?>> futures = new ConcurrentHashMap<>();
@@ -72,6 +75,7 @@ public class IndexFieldTypePollerPeriodical extends Periodical {
                                           final MongoIndexSet.Factory mongoIndexSetFactory,
                                           final Cluster cluster,
                                           final EventBus eventBus,
+                                          final ServerStatus serverStatus,
                                           @Named("index_field_type_periodical_interval") final com.github.joschi.jadconfig.util.Duration periodicalInterval,
                                           @Named("daemonScheduler") final ScheduledExecutorService scheduler) {
         this.poller = poller;
@@ -80,11 +84,14 @@ public class IndexFieldTypePollerPeriodical extends Periodical {
         this.indices = indices;
         this.mongoIndexSetFactory = mongoIndexSetFactory;
         this.cluster = cluster;
+        this.serverStatus = serverStatus;
         this.periodicalInterval = periodicalInterval;
         this.scheduler = scheduler;
 
         eventBus.register(this);
     }
+
+    private static final Set<Lifecycle> skippedLifecycles = ImmutableSet.of(Lifecycle.STARTING, Lifecycle.HALTING, Lifecycle.PAUSED, Lifecycle.FAILED, Lifecycle.UNINITIALIZED);
 
     /**
      * This creates index field type information for each index in each index set and schedules polling jobs to
@@ -95,6 +102,12 @@ public class IndexFieldTypePollerPeriodical extends Periodical {
      */
     @Override
     public void doRun() {
+        final Lifecycle currentLifecycle = serverStatus.getLifecycle();
+        if (skippedLifecycles.contains(currentLifecycle)) {
+            LOG.debug("Server is not running, skipping run.");
+            return;
+        }
+
         if (!cluster.isConnected()) {
             LOG.info("Cluster not connected yet, delaying index field type initialization until it is reachable.");
             while (true) {

--- a/graylog2-server/src/test/java/org/graylog2/indexer/fieldtypes/IndexFieldTypePollerPeriodicalTest.java
+++ b/graylog2-server/src/test/java/org/graylog2/indexer/fieldtypes/IndexFieldTypePollerPeriodicalTest.java
@@ -1,0 +1,69 @@
+/**
+ * This file is part of Graylog.
+ *
+ * Graylog is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Graylog is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Graylog.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.graylog2.indexer.fieldtypes;
+
+import com.github.joschi.jadconfig.util.Duration;
+import com.google.common.eventbus.EventBus;
+import org.graylog2.indexer.MongoIndexSet;
+import org.graylog2.indexer.cluster.Cluster;
+import org.graylog2.indexer.indexset.IndexSetService;
+import org.graylog2.indexer.indices.Indices;
+import org.graylog2.plugin.ServerStatus;
+import org.graylog2.plugin.lifecycles.Lifecycle;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.concurrent.ScheduledExecutorService;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class IndexFieldTypePollerPeriodicalTest {
+    private IndexFieldTypePollerPeriodical periodical;
+    private final IndexFieldTypePoller indexFieldTypePoller = mock(IndexFieldTypePoller.class);
+    private final IndexFieldTypesService indexFieldTypesService = mock(IndexFieldTypesService.class);
+    private final IndexSetService indexSetService = mock(IndexSetService.class);
+    private final Indices indices = mock(Indices.class);
+    private final MongoIndexSet.Factory mongoIndexSetFactory = mock(MongoIndexSet.Factory.class);
+    private final Cluster cluster = mock(Cluster.class);
+    @SuppressWarnings("UnstableApiUsage")
+    private final EventBus eventBus = mock(EventBus.class);
+    private final ServerStatus serverStatus = mock(ServerStatus.class);
+    private final ScheduledExecutorService scheduler = mock(ScheduledExecutorService.class);
+
+    @BeforeEach
+    void setUp() {
+        this.periodical = new IndexFieldTypePollerPeriodical(indexFieldTypePoller,
+                indexFieldTypesService,
+                indexSetService,
+                indices,
+                mongoIndexSetFactory,
+                cluster,
+                eventBus,
+                serverStatus,
+                Duration.seconds(30),
+                scheduler);
+    }
+
+    @Test
+    void executionIsSkippedWhenServerIsNotRunning() {
+        when(serverStatus.getLifecycle()).thenReturn(Lifecycle.HALTING);
+        when(cluster.isConnected()).thenThrow(new RuntimeException("If this exception is thrown, then execution is not skipped!"));
+
+        this.periodical.doRun();
+    }
+}

